frigate/log.py

```diff
@@ -35,7 +35,7 @@ def log_process(log_queue):
     while True:
         try:
             record = log_queue.get(timeout=5)
-        except queue.Empty:
+        except (queue.Empty, KeyboardInterrupt):
```

Comment:
Similar to above, an exception was being raised here on shutdown, so handling it here.